### DISCUSSION TITLE
[Chapter 08] `비영속`과 `준영속` 그리고 `영속`

### DISCUSSION THREAD

**Discussion by Irisation23** [cowaine/spring-boot-for-msa#127] [Chapter 08. Data Persistence] [answered]
해당 파트를 학습 하던 중 두가지 의구점이 생겼습니다.

### Q2. `준영속`을 다시 `영속`으로 변경하는 경우의 예제는 어떤 경우가 있을까?

보다 깊은 JPA 학습이 하고 싶어지는 계기가 되었던 [Chapter 08] 이였습니다.

**Comment by corock** [comment, marked as answer]
질문 1, 2번에 대해 탐구해 보려고 했으나... 때로는 전문가의 코멘트가 유용할 수 있기 때문에 공유합니다.

![yhkim-reply](https://github.com/cowaine/spring-boot-for-msa/assets/38161720/4bc9fc72-1bf5-41d2-b0c5-96b4fee3176f)
